beast/physicsmodel/prior_weights_dust.py

```diff
@@ -85,6 +85,23 @@ def _two_lognorm(xs,
     normalization = np.trapz(pointwise, x=xs)
     return pointwise / normalization
 
+def _exponential(x, a=2.0, N=1.):
+    """
+    Exponential distribution
+    Parameters
+    ----------
+    x: vector
+       x values
+    a: float
+       Single parameter in exp: a*e^-ax
```

Comment:
This should be N*e^-ax.

Reply:
I corrected the notation, but am leaving the param as "a" -- could use "decay_rate" or "decay_param" but that seems overkill.  Instead, I added additional text in docstring.